pkg/remoteconfig/state/agent_config.go

```diff
@@ -26,7 +26,8 @@ type AgentConfig struct {
 
 // ConfigContent contains the configurations set by remote-config
 type ConfigContent struct {
-	LogLevel string `json:"log_level"`
+	LogLevel         string `json:"log_level"`
+	EnableStreamLogs *bool
```

Comment:
It looks like this implies that there are going to be changes to the `AGENT_TASK` remote config product (and potentially the UI, to let the user toggle whether they want to enable streaming logs during flare collection?) If you haven't already, you should reach out to the Fleet Automation team at #fleet-automation, and include them in this review since they own the flare collection via RC 😄